Add localisation file for reflinks

zhaofengli opened the pull request:
Just a note that it's currently being used at https://tools.wmflabs.org/fengtools/reflinkstest/ ([Chinese version](https://tools.wmflabs.org/fengtools/reflinkstest/?userlang=zh)). It's still incomplete.

The localisation is implemented with a custom `msg()` function in the Twig templating engine which calls Intuition's `msg()` method. See [main.html](https://github.com/zhaofengli/reflinks/blob/master/style/templates/main.html) for an example.

Thank you.

Krinkle commented:
Looks good, but could you add `qqq` entries as well so translators know how the messages are used? And I noticed wiki is capitalised in your English messages, that should probably be changed to lowercase "wiki".

/cc @siebrand @Nikerabbit

zhaofengli commented:
Thanks, I've changed 'Wiki' to lowercase and squashed the commits. I've prefixed most of the keys with their corresponding UI elements (heading, button label, etc). Is executable code allowed, so that I can add notes in batches for those messages to describe where they will be used?

zhaofengli commented:
Hi, is this PR still being handled?

siebrand commented:
We will look into it Monday or Tuesday.

> Op 23 jan. 2015 om 02:57 heeft Zhaofeng Li <notifications@github.com> het volgende geschreven:
> 
> Hi, is this PR still being handled?
> 
> —
> Reply to this email directly or view it on GitHub.
>

siebrand commented:
qqq should be added for all messages in Reflinks.i18n.php. See https://www.mediawiki.org/wiki/I18n#Message_documentation for details.

zhaofengli commented:
Okay, no problem!

zhaofengli commented:
I've added the documentations.

zhaofengli commented:
By the way, since the main copy of the message file is in [the tool's repo](https://github.com/zhaofengli/reflinks/blob/master/src/Reflinks.i18n.php), I'm a little bit concerned about the workflow. What's the best way to keep them in sync and how can I get new messages on Translatewiki as soon as possible? Currently, Intuition is added as a git submodule in my repo, with a bootstrap script that [loads the message file manually](https://github.com/zhaofengli/reflinks/blob/master/src/bootstrap.php#L12). The tool must be usable off-Labs.

Krinkle commented on language/messages/Reflinks.i18n.php:
```diff
@@ -0,0 +1,293 @@
+	// General messages
+	/*
+		Well, I remember the the translation interface in Launchpad gives an automatic hint about how to deal with the parameters.
+		Maybe the Translate extension can do the same? (Or am I missing something?)
```
It already does. Messages are inspected while they type. Omitting parameters yields a warning. The convention is to describe the message and how it is used. Optionally followed by a list of parameters and what they will contain. There's no need to assume the user will remove it, or to explain "run-time" concepts. Translators see those all the time (though using "run-time" probably adds more confusion then it helps avoid).

> Please keep the $1 intact. It\'ll be replaced by a number at run-time.

May be better phrased as:

> Indicates how many references were modified to have citations.
> * $1: The number of fixed references.
